### ISSUE TITLE
[frontend] alt text shows above cw blackout

### ISSUE THREAD

**Issue by fluffye-yay** [gabboman/wafrn#253]
![Screenshot_20250107-180238](https://github.com/user-attachments/assets/dae38b46-6d87-4cc9-8eeb-97f4bebd1d4a)

**Comment by FireIsGood** [comment]
Is this still an issue after #255?

**Comment by fluffye-yay** [comment]
havent seen it so probably not
